server/database/queries/cohort/postCohort.js

```diff
@@ -0,0 +1,8 @@
+const connection = require('../../config/connection');
+
+exports.postCohort = (reqData) => {
+  return connection.query(
+    'INSERT INTO cohort (name, description, img_url, github_link) VALUES ($1, $2, $3, $4) RETURNING id;',
+    [reqData.cName, reqData.cDescription, reqData.cImgUrl, reqData.cGithub],
```

Comment:
Let the variable names be the same as in the database schema but using camelCase instead of underscore, and don't forget to make destructuring ``` reqData ```
```suggestion
    [reqData.name, reqData.description, reqData.imgUrl, reqData.githubLink],
```